docs/leap_sapi/sapi_rest.rst

```diff
@@ -667,7 +693,27 @@ query.\ [#]_
         params          Solver-specific
                         :ref:`hybrid parameters <opt_index_properties_parameters>`
                         or :ref:`QPU parameters <qpu_solver_parameters>`.
-        solver          Solver to be used.
+        solver          Unique identifier (name) of the solver to be used. For
+                        resource representation ``version=2.1`` and earlier, use
+                        the name of the solver (e.g., ``"solver": "solver_id"``).
+                        The ``version=2.1`` resource representation is
+                        deprecated; instead, use the ``version=3`` JSON
+                        structure. For resource representation ``version=3``,
+                        the solver identifier is represented by the following
+                        JSON structure::
+
+                        {"name": solver_name, "version": {"graph_id": graph_id}}
+
+                        where the fields are as follows:
+
+                        *    ``name``: Name of the solver.
```

Comment:
Either change this to ``solver_name``, or change the above JSON to use ``name`` for solver name.

Reply:
> Either change this to `solver_name`, or change the above JSON to use `name` for solver name.

Also notice that line 698 has this same value as `solver_id`

Reply:
Line 698, "For resource representation ``version=2.1`` and earlier, use the name of the solver (e.g., ``"solver": "solver_id"``)", means the user would set `"solver": "Advantage_system4.1"`, for example; Line 709 would be `{"name": "Advantage_system4.1", "version": {"graph_id": "01d07086e1"}}`, for example. 
So for the same value, "Advantage_system4.1", in 698 you have `solver_id` and in 709 you have `name`